Sources/PackageCollectionJSONModel/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2020-2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+add_library(PackageCollectionJSONModel
+  JSONPackageCollection.swift
+  JSONPackageCollection+v1.swift)
+
+set_target_properties(PackageCollectionJSONModel PROPERTIES
+  INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_Swift_MODULE_DIRECTORY})
+
+if(USE_CMAKE_INSTALL)
+install(TARGETS PackageCollectionJSONModel
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin)
+endif()
+set_property(GLOBAL APPEND PROPERTY SwiftPM_EXPORTS PackageCollectionJSONModel)
```

Comment:
Is this `set_property` call needed? I see it in some CMakeLists but not all.

Reply:
Not sure, paging @compnerd as the CMake expert